app/views/kata/_file_new_rename_delete.html.erb

```diff
@@ -83,6 +83,7 @@ $(function() {
     var scrollTop = oldFile.scrollTop();
     var scrollLeft = oldFile.scrollLeft();
     var caretPos = oldFile.caret();
+
     $(oldFile).closest('tr').remove();
```

Comment:
I'm not convinced this is correct, it leaves a div containing a table in the DOM.  I think it should be:

cd.fileDiv(oldFile).remove();

This isn't something from the changes in this pull request